docs/docs/v2v-migration-guide.md

```diff
@@ -0,0 +1,233 @@
+
+## 🎁 Benefits
+
+### In short
+
+The new V2V backend replaces the old approach, and delivers:
+
+- Warm migration for all ESXi versions
+- Improved performance
+- A cleaner design for easier maintenance
+
+### Detailed benefits
+
+#### Ease of use and compatibility
+
+The new backend only reads allocated blocks, which accelerates transfers, and snapshots have minimal impact on performance. There’s no longer any need for temporary NFS storage or remote VSAN targets. The system is also built to handle very large disks, with support for sizes beyond 2 TB planned in a future update.
+
+#### UI improvements
+
+The interface has also seen significant improvements. A progress bar is now visible from the start of migration, speed metrics are recorded in the VDI, and snapshots are created at each step. This allows you to pause and resume the process safely without creating multiple VMs. Additionally, UEFI VMs are no longer forced into Secure Boot mode, resolving previous boot issues.
+
+#### Performance gains 
+
+Performance gains can be significant, though results depend on the environment:
+
+- **In the best case**, when using VMs with many snapshots or mostly empty disks, migrations can be up to 100 times faster. In our high-performance lab, we measured around 150 MB/s per disk and up to 500 MB/s total, which means an infrastructure with 10 TB of data could be migrated in a single day, with less than five minutes of downtime per VM.
+A test case of importing a Debian VM with around 30GiB disk.
+
+- **In less favorable situations**, such as a fully allocated disk with no snapshots and a powered-off VM, improvements are smaller, mainly due to compression between XO and ESXi. In general, the limiting factor is the import speed on the XCP-ng side, which scales well until the storage is saturated. Most of the transfer occurs while the VM is running, so production data remains safe. 
+
+## 👨‍🍳 Preparing the XO environment
+
+### Update Xen Orchestra 
+
+Mmake sure Xen Orchestra is up to date. The import page now includes an automatic check that highlights errors and warnings before migration. Errors must be resolved, and warnings should be addressed when possible:
+
+The new setup completly replace the previous solution. An auto check is added at the top of the import page. Any error must be handled before trying a migration. Any warning should be addressed if possible. 
+<img width="481" height="239" alt="image" src="https://gist.github.com/user-attachments/assets/8ebda4cc-fe11-4bfb-ac6e-d1e6025c023d" />
+
+### Dependencies
+
+1. Install the required dependencies: `nbdkit`, the VDDK plugin, and `nbdinfo`.
+
+:::tip
+On Debian, these are available through the contrib repository as `nbdkit-plugin-vddk` and `libnbd-bin`. You can also compile `nbdkit` and `libnbd` from source on GitLab, but the correct dependencies must be in place for `nbdinfo` and VDDK.
+:::
```

Comment:
this PR improve the process : https://github.com/vatesfr/xen-orchestra/pull/8927 

the contrib version is too old , even with debian 13, but there now button that will download, compile and install the packages. This part is only tested on debian 12/13, and need to have access to internet. 

In the future we should be able to distribute the XOA with the library up to date